### PR TITLE
Merge Straight to Main

### PR DESCRIPTION
**Pull request by Missive00**
This where I stop?

### CODE REVIEW

File: Assets/JL_MouseOverText.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class JL_MouseOverText : MonoBehaviour
+{
+    // Start is called before the first frame update
+
+[SerializeField] string MouseOverText;
```
Comment (Thalpy) [inline Assets/JL_MouseOverText.cs:9]: Why no indent

File: Assets/Scripts/GameMaster.cs [hunk 2]
```diff
@@ -13,7 +13,7 @@
 [RequireComponent(typeof(CrisisMaster))]
 [RequireComponent(typeof(CardMaster))]
 [RequireComponent(typeof(FactionController))]
-[RequireComponent(typeof(JL_HandController))]
+//[RequireComponent(typeof(JL_HandController))]
```
Comment (Thalpy) [inline Assets/Scripts/GameMaster.cs:16]: Just remove it

File: Assets/JL_MouseOverText.cs (hunk 1 above)
Comment (apapiez) [reply Assets/JL_MouseOverText.cs:9]: THIS IS UNDEED UPSETTING, ISHNU ALA

File: Assets/Scripts/GameMaster.cs (hunk 2 above)
Comment (apapiez) [reply Assets/Scripts/GameMaster.cs:16]: Agreed

File: Assets/JL_MouseOverText.cs (hunk 1 above)
Comment (Thalpy) [reply Assets/JL_MouseOverText.cs:9]: ISHNU AL DIEM

**Review by apapiez — APPROVED** [review]
A couple of bits to consider, as per comments but shouldn't be any breaking conflicts

File: Assets/JL_MouseOverText.cs (hunk 1 above)
Comment (apapiez) [reply Assets/JL_MouseOverText.cs:9]: NEED MORE RAGE

**Pull request merged**